metrics/cuad/evaluate.py

```diff
@@ -0,0 +1,207 @@
+""" Official evaluation script for CUAD dataset. """
```

Comment:
Did you have to do some modifications to this script ?
If so please add comments to show the lines you modified

Reply:
Most the things were kept same (as reported in the paper). Only `exact_match` metric was added